packages/react-components/src/components/TogetherModeOverlay.tsx

```diff
@@ -177,6 +176,7 @@ export const TogetherModeOverlay = memo(
             participantStatus.id && (
               <div
                 key={participantStatus.id}
+                data-ui-group="together-mode-participant"
```

Comment:
For our hermetic and unit tests we usually use `data-ui-id` unsure what the group is and how it would impact our existing selector fetching. if you want to reuse the utilities in the hermetic tests you will want to use this prop instead